### DISCUSSION TITLE
Unable to type value in plugins

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#1207] [Help]
I noticed something weird for a long time now : 
Whenever I try to type a value in a plugin field (to set a value in db for example), it doesnt type the correct characters !
So I am unable to type something like 1.5.Digits are typed as something else. . A space become a dot etc ...
Tested on two Mac M1 (client/server running monterey) with a remote server.
A bit annoying ...

**Comment by apohl79** [comment]
Wow, never saw that. What keyboard layout are you using (even though the code should be layout agnostic)?

Is that happen anywhere? Like also in the Apple AUs etc?

**Comment by Henri75** [comment]
Happens with french and azerty, Magic keyboard with numeric pad.
Doesnt seem to be directly related to the layout, as I am unable to type
numbers !
I will check, but from my memory it doesnt happen on windows remote server

Le mer. 3 mai 2023 à 10:33, Andreas Pohl ***@***.***> a
écrit :
